src/goodHealthApp/urls.py

```diff
@@ -1,12 +1,13 @@
 from django.urls import path, include
 from rest_framework import routers
 
-from .views import UserPostViewSet, UserProfileViewSet, UserViewSet
+from .views import UserPostViewSet, UserProfileViewSet, UserViewSet, EmergencyLineViewset
 
 router = routers.DefaultRouter()
 router.register(r'userprofiles', UserProfileViewSet)
 router.register(r'postfeed', UserPostViewSet)
 router.register(r'users', UserViewSet)
+router.register(r'emergencyline', EmergencyLineViewSet)
```

Comment:
Change `router.register(r'emergencyline', EmergencyLineViewSet)` to `router.register(r'emergencylines',EmergencyLineViewSet)`
The most commonly used practice for naming APIs is plural, but asides from that I just think you should change that line so that the new commit will delete it and re-add it to the file to try rectifying the codacy errors from this file